Sound/Tidal/Pattern.hs

```diff
@@ -700,6 +704,13 @@ almostAlways = sometimesBy 0.9
 never = flip const
 always = id
 
+{- | `somecyclesBy` is a cycle-by-cycle version of sometimesBy, and has a
```

Comment:
replace backticks with `@` for highlight (and I think referencing) of functions. also `sometimesBy` should be marked I guess.